### PR TITLE
Update benchmarks.md - attempt 2.0

### PR DESCRIPTION
**Pull request by hennyg888**
I created a clean branch from the latest version of master that passed the documentation buildkite test. Looking at my old updated `benchmarks.md` through a markdown viewer/editor and I saw that a table that's written in HTML might be causing trouble so I replaced it with a code block quote table.

### CODE REVIEW

**Comment by glwagner** [comment]
Have you tried building the docs locally to see how the new `benchmarks.md` looks?

**Comment by hennyg888** [comment]
How would I build the docs locally? Would I just run `docs/make.jl`?

**Comment by glwagner** [comment]
> How would I build the docs locally? Would I just run `docs/make.jl`?

I summon @navidcy

**Comment by navidcy** [comment]
The instructions [in the docs](https://clima.github.io/OceananigansDocumentation/stable/contributing/#Documentation) don't do the job?

File: docs/src/appendix/benchmarks.md [hunk 1]
```diff
@@ -1,12 +1,673 @@
-# [Performance benchmarks](@id performance_benchmarks)
+
+# Performance benchmarks
```
Comment (navidcy) [inline docs/src/appendix/benchmarks.md:2]:
```suggestion
# [Performance benchmarks](@id performance_benchmarks)
```

**Review by navidcy — CHANGES_REQUESTED** [review]
I'm not 100% sure but I feel you need this *since* you are making a reference to it at

https://github.com/CliMA/Oceananigans.jl/blob/f1250ddb2a7a06b64b0fcfaee7eafa23f0dd0e19/docs/src/using_gpus.md#L6

**Comment by hennyg888** [comment]
Thank you very much @navidcy! Docs generated and all checks passed.

**Pull request merged**